test/screens/v2/widget_instance/departures_test.exs

```diff
@@ -44,6 +44,21 @@ defmodule Screens.V2.WidgetInstance.DeparturesTest do
                Departures.serialize_section(section, bus_shelter_screen, now)
     end
 
+    test "returns serialized normal_section with destination grouping", %{
```

Comment:
Since it doesn't have any rows, this test case doesn't seem like it's doing much other than making sure this code path doesn't crash outright, which the full test case on L288 also does. Could we remove it without losing anything?